Jenkinsfile

```diff
@@ -19,6 +19,13 @@ elifePipeline {
             stage 'Merge to master', {
                 elifeGitMoveToBranch commit, 'master'
             }
+
+            stage 'Push unstable image', {
+                def image = DockerImage.elifesciences(this, 'sciencebeam-trainer-grobid', commit)
```

Comment:
This won't work because we also need to include the `GROBID_TAG` in the image tag.